app/controllers/miq_ae_class_controller.rb

```diff
@@ -1723,6 +1682,10 @@ def refresh_git_domain
     replace_right_cell(:replace_trees => [:ae])
   end
 
+  def namespace
+    render :json => find_record_with_rbac(MiqAeNamespace, params[:id]).attributes.slice('name', 'description', 'enabled')
```

Comment:
Shouldn't `assert_privileges()` come before this?

Reply:
Indeed. Added :)